.travis.yml

```diff
@@ -7,6 +7,9 @@ node_js:
   - 8
   - 10
 
+before_install:
+  - dpkg --compare-versions `npm -v` ge 6 || npm i -g npm@^6
```

Comment:
Just realized that it _might_ be a good idea to use the older npm when testing Node.js 6, since that is what people using Node.js 6 will be using. That way we would catch errors/incompatibilities with older npm versions.

We would still get the benefit of getting the `package-lock.json` checked since `npm ci` would be used in the newer Node.js versions.

Not a big deal though, I'm fine either way. Could be nice with a bit less code though ☺️

Reply:
Hmm, just read the initial PR description as well, this is probably good then 👍